docs/debugging/providers_and_languages.md

```diff
@@ -0,0 +1,19 @@
+# Providers and Languages
+
+Languages and providers run as their own executables, so debugging isn't as
+simple as attaching to the engine.
+
+Instead you'll have to launch them in a debugger and provide the information
+for connecting to them to the engine via environmental variables.
+
+## Languages
+
+Languages work very similary to [Providers](#Providers).
```

Comment:
```suggestion
Languages work very similarly to [Providers](#Providers).
```

Reply:
Where does `#Providers` go? Is Sphinx case-insensitive and it goes to the `providers` section?